spec/requests/api/v1/github_webhook/filter_spec.rb

```diff
@@ -17,6 +18,16 @@
     }
   end
 
+  let!(:available_channels) do
+    ['javascript-code-review', 'react-code-review', 'react-native-code-review',
```

Comment:
you can get the real available channels using the map function and SlackNotificationService::LANGUAGES + '-code-review'

Reply:
I don't know why this still appears but I changed it some commits ago